lib/Controller/PageController.php

```diff
@@ -306,19 +294,9 @@ public function validationFileWithShortUrl(): RedirectResponse {
 	#[PublicPage]
 	#[RequireSignRequestUuid]
 	public function resetPassword(): TemplateResponse {
-		$this->initialState->provideInitialState('config', array_merge(
+		$this->initialState->provideInitialState('config',
 			$this->accountService->getConfig($this->userSession->getUser()),
```

Comment:
Can remove the comma on final of the line

Reply:
At row 297? If yes, isn't possible. If are you talking about row 298, yes, is possible.

Reply:
Yes, I was talking about 298 line.